apps/api/src/app/notification-groups/dtos/delete-notification-group-response.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsBoolean, IsDefined, IsString } from 'class-validator';
+
+export class DeleteNotificationGroupResponseDto {
+  @ApiProperty({
+    description: 'A boolean stating the success of the action',
+  })
+  @IsBoolean()
+  @IsDefined()
+  acknowledged: boolean;
+
+  @ApiProperty({
+    description: 'The status enum for the performed action',
+    enum: ['deleted'],
+  })
+  @IsString()
+  @IsDefined()
+  status: string;
```

Comment:
What's the reason behind adding this status to the API response of a call that from the method we know it is a delete call? Did you have something in mind that in the future we could benefit from having a status?

Reply:
I got this from the issue, @scopsy mentioned it somewhere. Also, we have similar in the remove-subscribe usecase

Reply:
I am not particularly fan of returning 200 status for successful delete calls as I consider it complicates consumer's logic besides not adding extra useful information. As it is an implementation detail requested I will discuss it in the core team. 🙂

Reply:
Alright then.

Unrelated, but I admired your eye for detail and thorough reviews, especially for the PRs I have raised so far. It helps me to understand the codebase better and overall, a skill I want to emulate.